diff --git a/lede/target/linux/rockchip/armv8/base-files/etc/board.d/02_network b/lede/target/linux/rockchip/armv8/base-files/etc/board.d/02_network
--- a/lede/target/linux/rockchip/armv8/base-files/etc/board.d/02_network
+++ b/lede/target/linux/rockchip/armv8/base-files/etc/board.d/02_network
@@ -10,7 +10,7 @@ rockchip_setup_interfaces()
	case "$board" in
	friendlyarm,nanopi-r2s|\
	friendlyarm,nanopi-r4s)
-		ucidef_set_interfaces_lan_wan 'eth1' 'eth0'
+		ucidef_set_interfaces_lan_wan 'eth0' 'eth1'
		;;
	*)
		ucidef_set_interface_lan 'eth0'
